pkg/test/olm/che_cluster.go

```diff
@@ -41,6 +42,12 @@ func (a *CheClusterAssertion) HasRunningStatus(want string) *CheClusterAssertion
 	return a
 }
 
+func (a *CheClusterAssertion) HasServerURL(want string) *CheClusterAssertion {
```

Comment:
This is not used anywhere. Maybe you just forgot to add the call of this method, but wouldn't be too hard to verify the specify URL? I mean - wouldn't be too hard to create/get the expected URL. 
Wouldn't be easier to verify that the `cheCluster.Status.CheURL`  is not empty and maybe that it contains some expected substrings? This is up to you of course ;-)

Reply:
It's a mistake. I will use the `CheInstallationAssertion.HasServerURL(string)` method instead

Reply:
removed in ee8c6ab